llmserve/backend/llm/pipelines/llamacpp/llamacpp_pipeline.py

```diff
@@ -218,3 +224,44 @@ def from_initializer(
             device=device,
             **kwargs,
         )
+
+    def streamGenerate(self, prompt: str, **generate_kwargs) -> Generator[str, None, None]:
+        logger.info(f"stream prompt: {prompt}")
+        inputs = construct_prompts(prompt, prompt_format=self.prompt_format)
+        logger.info(f"stream inputs: {inputs}")
+        
+        try:
+            inputs_bak = inputs
+            inputs = [json.loads(prompt, strict=False) for prompt in inputs]
+        except Exception as ex:
+            logger.error(f"Exception apply_chat_template: {ex}")
+            logger.info("Seems no chat template from user")
+            inputs = inputs_bak
+            
+        logger.info(f"Forward generate_kwargs: {generate_kwargs}")
+        logger.info(f"model inputs: {inputs}")
+        
+        streamer = TextIteratorStreamer(self.tokenizer, timeout=0, skip_prompt=True, skip_special_tokens=True)
+        # input_ids = self.tokenizer([prompt], return_tensors="pt")
+        input_ids = self.tokenizer(inputs, return_tensors="pt")
+        max_new_tokens = 256
+        if generate_kwargs["max_tokens"]:
+            max_new_tokens = generate_kwargs["max_tokens"]
+        generation_kwargs = dict(input_ids, streamer=streamer, max_new_tokens=max_new_tokens)
+        thread = Thread(target=self.model.generate, kwargs=generation_kwargs)
```

Comment:
The `model.generate` will not work as expected, since `llamacpp python` lib have `model.create_chat_completion` to apply chat template

Reply:
change to create_chat_completion